content/layouts/page-react.html

```diff
@@ -18,7 +18,7 @@ <h4>{{ maintenance_line2 }}</h4>
           </div>
         </div>
       </div>
-  {% else %}
+  {% elsif custom_react_root != true %}
```

Comment:
This exists because we need to put the beta button in a specific spot and the react template just appends it to the end of the page. I think this changed in between now and the last beta.